plugins/modules/template_intent.py

```diff
@@ -2175,13 +2183,13 @@ def requires_update(self):
         """
 
         if self.have_template.get("isCommitPending"):
-            self.log("Template is in saved state and needs to be updated and committed")
+            self.log("Template is in saved state and needs to be updated and committed.", "DEBUG")
```

Comment:
Template name?
Template(template Name) is in a saved state and requires updating and committing.